management/src/test/java/org/ehcache/docs/ManagementTest.java

```diff
@@ -71,9 +72,16 @@ public void usingManagementRegistry() throws Exception {
 
     Map<String, String> context = createContext(managementRegistry); // <5>
 
-    Collection<Counter> counters = managementRegistry.collectStatistics(context, "StatisticsCapability", "GetCounter"); // <6>
+    Collection<Statistic<?, ?>> counters = managementRegistry.withCapability("StatisticsCapability") // <6>
```

Comment:
I don't find the following DSL to be particularly readable. build().execute().get(0).getStatistics() gives me a felling that I have no idea what kind of objects are being returned in between calls.